download/download.go

```diff
@@ -1,21 +1,56 @@
 package download
 
 import (
+	"encoding/json"
 	"fmt"
 	"log"
 	"net/http"
+	"os"
+	"path/filepath"
 	"sort"
 	"time"
 )
 
+func createLastUpdateJSONFile(dir string, dt []string) error {
+	// Create last_update.txt
+	f, err := os.Create(filepath.Join(dir, "last_update.txt"))
+	if err != nil {
+		return err
```

Comment:
No projeto sempre “envelopamos” os erros com uma mensagens que dão um contexto sobre eles, por exemplo: 

```suggestion
	pth := filepath.Join(dir, "last_update.txt")
	f, err := os.Create(pth)
	if err != nil {
		return fmt.Errorf("error creating last update file at %s: %w", pth, err)
```